rust/src/ike/ikev1.rs

```diff
@@ -40,27 +40,27 @@ pub struct IkeV1Header {
 pub struct Ikev1ParticipantData {
     pub key_exchange: String,
     pub nonce: String,
-    pub vendor_ids: HashSet<String>,
-    /// nested Vec, outer Vec per Proposal/Transform, inner Vec has the list of attributes.
-    pub transforms: Vec<Vec<SaAttribute>>,
+    pub nb_transforms: u64,
```

Comment:
so what happens if we actually get some really high number in here? the u64 suggests we'd handle it but given the loops of the actual transforms vec I'm not so sure

Reply:
To reach it, someone needs to send at least U64_MAX bytes of data...

The worst that would happen then, is that we would stop logging the transform chosen by the server (which is controlled by the attacker in this scenario)